src/parser/diffeq.yy

```diff
@@ -40,7 +40,7 @@
 
 /** add extra arguments to yyparse() and yylexe() methods */
 %parse-param {class DiffeqLexer& scanner}
-%parse-param {diffeq::DiffEqContext& context}
+%parse-param {diffeq::DiffEqContext& eq_context}
 %lex-param   {diffeq::DiffEqScanner &scanner }
```

Comment:
not sure why this is appearing here because other PR is merged to master already. let me check this.